sorc/grid_tools.fd/regional_esg_grid.fd/pmat.f90

```diff
@@ -42,69 +44,72 @@ module pmat
 
 contains
 
-!=============================================================================
+!> Swap vectors
+!!
+!! @param d vector
```

Comment:
Always use @param[in] @param[out] or @param[inout]

Reply:
done